modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,93 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
```

Comment:
Why is there an extra line of whitespace here?

Reply:
Can't find a whitespace here, please double check.

Reply:
That extra line is on every file in that folder. I think it should be there, see here an example from another file in that folder:

https://github.com/octobercms/october/blob/2aac294a5b74c4a219bf2078088c89f087776e9e/modules/backend/assets/js/october.sidenav.js#L138-L140

Reply:
@LukeTowers Please could you double check this, I think there should be a whitespace line here, thanks.

Reply:
@ayumihamsaki fair enough